src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
```

Comment:
Please remove this comment. Comments should not appear in the code except Java docs and some moments when this comment is an explanation for some code.